api/lib/source/polkadotV0-source.js

```diff
@@ -1214,10 +1216,7 @@ class polkadotAPI {
       ),
       topVoters: await Promise.all(
         topVoters.map(async (topVoterAddress) => {
-          const accountInfo = await this.getNetworkAccountInfo(
-            topVoterAddress,
-            api
-          )
+          const accountInfo = await api.derive.accounts.info(topVoterAddress)
```

Comment:
didn't we discuss, that here we can also use `getNetworkAccountInfo`?

Reply:
true, I don't know what happened here. Probably was changed in develop but the original PR this one came from was older. Thank you for remembering this